packages/firestore/exp/test/deps/dependencies.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Note: This file is generated by the Node CLI tool and then used in the unit test as the source of truth. It is checked in and as such provides a nice diff as we make changes.